packages/server/lib/controllers/oauth.controller.ts

```diff
@@ -379,13 +379,24 @@ class OAuthController {
                     oauth2Client.getSimpleOAuth2ClientConfig(providerConfig, template, connectionConfig)
                 );
 
-                const authorizationUri = simpleOAuthClient.authorizeURL({
+                let authorizationUri = simpleOAuthClient.authorizeURL({
                     redirect_uri: callbackUrl,
                     scope: providerConfig.oauth_scopes ? providerConfig.oauth_scopes.split(',').join(oauth2Template.scope_separator || ' ') : '',
                     state: session.id,
                     ...allAuthParams
                 });
 
+                if (template.authorization_url_replacements) {
+                    const urlReplacements = template.authorization_url_replacements || {};
+
+                    Object.keys(template.authorization_url_replacements).forEach((key) => {
+                        const replacement = urlReplacements[key];
+                        if (replacement !== undefined && typeof replacement === 'string') {
```

Comment:
isn't `typeof replacement === 'string'` already ensure it won't be undefined

Reply:
Whoops, I updated this in code but didn't push the update. Updating

Reply:
Updated with https://github.com/NangoHQ/nango/pull/1673/commits/112b2584c8e87326ca73218c4bd5cbdf27e7a088